EvidenceApi/V1/UseCase/ResidentRequestValidator.cs

```diff
@@ -11,8 +9,9 @@ public class ResidentRequestValidator : AbstractValidator<ResidentRequest>, IRes
         public ResidentRequestValidator()
         {
             RuleFor(x => x.Name).NotEmpty();
-            RuleFor(x => x.Email).NotEmpty().EmailAddress();
-            RuleFor(x => x.PhoneNumber).NotEmpty();
+            RuleFor(x => x.Email).EmailAddress();
+            RuleFor(x => x.Email).NotEmpty().When(x => string.IsNullOrEmpty(x.PhoneNumber)).WithMessage("'Email' and 'Phpne number' cannot be both empty.");
+            RuleFor(x => x.PhoneNumber).NotEmpty().When(x => string.IsNullOrEmpty(x.Email)).WithMessage("'Email' and 'Phpne number' cannot be both empty.");
```

Comment:
Typo on these 2 lines
```suggestion
            RuleFor(x => x.PhoneNumber).NotEmpty().When(x => string.IsNullOrEmpty(x.Email)).WithMessage("'Email' and 'Phone number' cannot be both empty.");
```